pimd/pim_cmd_common.c

```diff
@@ -2832,6 +2832,8 @@ static int pim_print_json_pnc_cache_walkcb(struct hash_bucket *backet,
 	json_object *json_row = NULL;
 	json_object *json_ifp = NULL;
 	json_object *json_arr = NULL;
+	struct pim_interface *pim_ifp;
+	bool pim_enable;
```

Comment:
Some Coverity thing is going to mark the fact that pim_enable not set to an intial value will cause the undefined behavior on lines 2865 and 2869 and then we'll have to get someone to fix this.

Reply:
Thanks @donaldsharp.
I have fixed this.